src/routes/tftRoutes.js

```diff
@@ -5,34 +5,69 @@ import riotApiClient from '../services/riotApiClient'
 function getTFTRoutes(){
     const router = express.Router()
     router.get('/v1/summoner/:summonerName', getSummonerHandler)
-    router.get('/v1/match/:summonerName', getRecentMatchInfo)
-    router.get('/v1/match/history/:summonerName', getMatchHistoryForSummoner)
-    //router.get('/v1/stats/:count/:summonerName', getAverageScores)
+    router.get('/v1/match/history/:summonerName/:matchId', getMatchInfo)
+    router.get('/v1/matchId/list/:summonerName', getListOfMatchIds)
```

Comment:
Same here, use handleWhatever the handler does:
```suggestion
    router.get('/v1/matches/:summonerName', handleGetMatchesForSummoner)
```